packages/pymedphys_dicom/tests/test_anonymise.py

```diff
@@ -257,6 +257,66 @@ def test_anonymise_cli():
     finally:
         remove_file(test_anon_filepath)
 
+    # File anonymisation - preserve filenames
+    assert not is_anonymised_file(test_filepath)
+    expected_anon_filepath = anonymised_dicom_filepath(test_filepath,
+                                                       preserve_original=True)
+    assert not exists(expected_anon_filepath)
+    anon_file_pres_command = ('pymedphys dicom anonymise -f'.split()
+                              + [test_filepath])
+    try:
+        subprocess.check_call(anon_file_pres_command)
+        assert is_anonymised_file(expected_anon_filepath)
+        assert exists(test_filepath)
+    finally:
+        remove_file(expected_anon_filepath)
+
+    # File anonymisation - clear values
+    assert not is_anonymised_file(test_filepath)
+    assert not exists(test_anon_filepath)
+    anon_file_clear_command = ('pymedphys dicom anonymise -c'.split()
+                               + [test_filepath])
+    try:
+        subprocess.check_call(anon_file_clear_command)
+        assert is_anonymised_file(test_anon_filepath)
+        assert pydicom.dcmread(test_anon_filepath).PatientName == ''
+        assert exists(test_filepath)
+    finally:
+        remove_file(test_anon_filepath)
+
+    # File anonymisation - leave keywords unchanged
+    assert not is_anonymised_file(test_filepath)
+    assert not exists(test_anon_filepath)
+    anon_file_keep_command = ('pymedphys dicom anonymise'.split()
+                              + [test_filepath]
+                              + '-k PatientName'.split())
+    try:
+        subprocess.check_call(anon_file_keep_command)
+        assert not is_anonymised_file(test_anon_filepath)
+        ds = pydicom.dcmread(test_anon_filepath)
+        ds.PatientName = "Anonymous"
```

Comment:
Not for this pull request, but I think there would be value in learning how to use https://pydicom.github.io/pydicom/stable/ref_guide.html#pydicom.filereader.read_partial with the `stop_when` parameter. I think once we learn how to use that we could as a whole speed up tests that load DICOM files.

Reply:
Yep, sounds good :)